js/views/composerview.js

```diff
@@ -394,7 +404,7 @@ define(function(require) {
 					onSuccess();
 				}
 
-				if (this.draftUID !== null) {
+				if (_this.draftUID !== null) {
 					// update UID in message list
 					var collection = Radio.ui.request('messagesview:collection');
 					var message = collection.findWhere({id: this.draftUID});
```

Comment:
Actually, I continuously get a JS error while composing my emails:

> Uncaught (in promise) TypeError: Cannot read property 'draftUID' of undefined

But it does not matter here anyway, I think I already talked about that one in another issue about drafts...